Code/GraphMol/FMCS/testFMCS_Unit.cpp

```diff
@@ -596,55 +598,131 @@ void testSimpleFast() {
     BOOST_LOG(rdInfoLog) << "\tdone" << std::endl;
 }
 
-
-#ifdef RDK_TEST_MULTITHREADED
-namespace {
-  void runblock() {
+void testStereo() {
+    BOOST_LOG(rdInfoLog) << "-------------------------------------" << std::endl;
```

Comment:
This isn't actually testing anything. It's just running the FMCS code. Please set it up so that it actually tests the results of the FMCS calls.